src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_inline {
+  width: 16px;
+  height: 16px;
+  margin: 2px;
+  background: url(images/star.svg) no-repeat;
+  list-style-type: none;
+}

```

Comment:
You add margin to all sides, add only for the right side and remove margin in the last star

Reply:
Thanks, resolved